testing/scenario_app/ios/Scenarios/ScenariosUITests/TextSemanticsFocusTest.m

```diff
@@ -6,6 +6,36 @@
 
 FLUTTER_ASSERT_ARC
 
+@interface XCUIElement (ftr_waitForNonExistence)
+/// Keeps waiting until the element doesn't exist.  Returns NO if the timeout is
```

Comment:
keep waiting until the element no longe exists?

Reply:
That sounds like it has to exist beforehand.  The implementation is agnostic to that.